foundations/src/telemetry/log/init.rs

```diff
@@ -95,12 +96,29 @@ pub(crate) fn init(service_info: &ServiceInfo, settings: &LoggingSettings) -> Bo
         }
         (LogOutput::File(file), LogFormat::Json) => {
             let buf = BufWriter::with_capacity(BUF_SIZE, File::create(file)?);
-            let drain = build_json_log_drain(buf);
+            let drain = build_json_log_drain(buf, false);
             AsyncDrain::new(drain).chan_size(CHANNEL_SIZE).build()
         }
+        (LogOutput::Stdout, log_fmt) => {
```

Comment:
I think we don't need additional `LogOutput` type. `Terminal` is basically stdout